baseplate/context/kombu.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from kombu import Connection
+from kombu import Exchange
+from kombu.pools import Producers
+
+from baseplate import config
+from baseplate.context import ContextFactory
+
+
+def connection_from_config(app_config, prefix, **kwargs):
```

Comment:
thanks for adding these! they need docstrings and to be in the .rst file though :(